layouts/partials/openapi/render-object-table.html

```diff
@@ -9,11 +9,11 @@
   * `properties`: optional dictionary of the properties to list, each given as:
     `property_name` : `property_data`
 
-  * `additionalProperties`: optional dictionary for properties with undefined
-    names, in the same format as `property_data`
+  * `additionalProperties`: an OpenAPI schema document for additional properties
```

Comment:
The term "OpenAPI schema document" sounds weird to me. I have seen "OpenAPI Document" or "OpenAPI Definition", which are not right here. I believe that "JSON schema" is both correct here and easier to understand.

Reply:
Thanks. Addressed in f2ebb400.